Src/StartMenu/StartMenuDLL/StartMenuDLL.rc

```diff
@@ -1162,25 +1162,34 @@ BEGIN
     IDS_STRING7036          "Text color for first column (RRGGBB)#Select custom color for the first column of the main menu text"
     IDS_STRING7037          "Text color for second column (RRGGBB)#Select custom color for the second column of the main menu text"
     IDS_STRING7038          "Text size#Select custom size for the main menu text"
+    IDS_STRING7039          "Menu theme"
+    IDS_STRING7040          "Light"
+    IDS_STRING7041          "Dark"
+    IDS_STRING7042          "Use system setting"
+    IDS_STRING7043          "Transparency style"
+    IDS_STRING7044          "Two-tone#The main section of the menu will use system light/dark colors while additional surfaces use transparent glass with accent color."
+    IDS_STRING7045          "Full glass#The entire main menu will use transparent glass with accent color"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_STRING7100          "This is the default skin when no other skin is selected or if the selected skin fails to load.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7101          "Windows Aero skin\n\nDefault skin to use for the Windows Aero theme.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7102          "Windows Basic skin\n\nDefault skin to use for the Windows Basic theme.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7103          "Classic skin\n\nClassic look with large or small icons.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
+    IDS_STRING7100          "This is the default skin when no other skin is selected or if the selected skin fails to load.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
```

Comment:
These don't look like wanted changes.
It adds duplicated copyright string.

Reply:
Yes, I see. I solve it, but You are was first, ;-)